ExoPlayer.Core/ExoPlayer.Core.csproj

```diff
@@ -7,6 +7,7 @@
     <PackageId>Xam.Plugins.Android.ExoPlayer.Core</PackageId>
     <IsBindingProject>true</IsBindingProject>
     <EnableDefaultItems>false</EnableDefaultItems>
+    <!--<AndroidClassParser>class-parse</AndroidClassParser>-->
```

Comment:
Why is this changed?

Reply:
I think the SDK extras already set this by default. Please remove the whole line.